TILE_AUTHOR_GUIDE.md

```diff
@@ -0,0 +1,598 @@
+
+You can reference the part in base.yml by using the follwing template functions:
+- `bosh_variable` reads, interpolates, and formats a product template part from `./bosh_variables`
+- `form` reads, interpolates, and formats a product template part from `./forms`
+- `property` reads, interpolates, and formats a product template part from `./properties`
+- `release` reads, interpolates, and formats a BOSH release data from either the `./releases` directory or the Kilnfile.lock
+- `stemcell` reads, interpolates, and formats a Stemcell Criteria data from the Kilnfile.lock
+- `version` returns the contents of the `./version` file
+- `variable` finds the named variables from either a `--variables-file` or `--variable`
+- `icon` returns the base64 encoded content of `icon.png`
+- `instance_group` reads, interpolates, and formats a product template part from `./instance_groups`
+- `job` reads, interpolates, and formats a product template part from `./jobs`
+- `runtime_config` reads, interpolates, and formats a product template part from `./runtime_configs`
+
+Other functions:
+- `tile` _TODO function documentation_
+- `select`  _TODO function documentation_
+- `regexReplaceAll`  _TODO function documentation_
+
+See the [crhntr/hello-tile/base.yml](https://github.com/pivotal/hello-tile/blob/c6b59dcb1118c9b2f5d4fbf836842ce4033baa80/base.yml#L29C1-L30C1) for some use of the above functions.
+The property definition in [crhntr/hello-tile/properties/hello.yml](https://github.com/pivotal/hello-tile/blob/c6b59dcb1118c9b2f5d4fbf836842ce4033baa80/properties/hello.yml#L2-L5)
+in referenced in `base.yml` using `$( property "port" )`.
+Most other product template part functions behave similarly.
+
+## <a id="bosh-release-tarballs"></a> Managing BOSH Release Tarballs
+
+`kiln fetch` downloads BOSH Release Tarballs from any of the following "sources"
+and puts them in a "./releases" directory.
+
+Before Kiln can help you manage the BOSH Releases you put in the tile, you need to upload BOSH Release tarballs to a place accessable to Kiln.
+
+See the sources below to decide which is right for your release.
+
+Unlike Tile Generator.
+Kiln does not create releases.
+The Kiln way is to 
+- have BOSH releases each have their own source control
+- have CI build and upload (finalized) BOSH release tarballs somewhere
+- have a tile source repository
+  - in the tile source repository specify how Kiln can get the BOSH release tarballs
+
+While the following examples start from empty directories and mostly use S3 and BOSH.io.
+There are similar simple scripts for a small test tile illustrating similar usage patterns to the follwoing example.
+See [hello-tile](https://github.com/crhntr/hello-tile).
+Hello Tile consumes a single custom BOSH Release, [hello-release](https://github.com/crhntr/hello-release), from a GitHub release.
+It does not upload the release to PivNet but adds the built tile to a GitHub Release.
```

Comment:
Maybe change the reference from `PivNet` to `TanzuNet`?

Reply:
Yea... I'll switch to TanzuNet to be consistent.